packages/office-ui-fabric-react/src/components/FloatingPicker/Suggestions/SuggestionsControl.tsx

```diff
@@ -90,13 +91,22 @@ export class SuggestionsControl<T> extends BaseComponent<ISuggestionsControlProp
   }
 
   public render(): JSX.Element {
-    const { className, headerItemsProps, footerItemsProps } = this.props;
+    const { className, headerItemsProps, footerItemsProps, suggestionsAvailableAlertText } = this.props;
+
+    const screenReaderTextStyles = mergeStyles(hiddenContentStyle);
```

Comment:
Nit, pull this out, since screenreadertextstyles won't change ever we should only call mergeStyles once.